storage/src/cache/cachedfile.rs

```diff
@@ -51,33 +51,44 @@ impl FileCacheMeta {
         blob_file: String,
         blob_info: Arc<BlobInfo>,
         reader: Option<Arc<dyn BlobReader>>,
+        sync: bool,
```

Comment:
How about abstracting the `sync` related code as a new function `try_open_cached_xxx()`?

Reply:
But we still need to download the meta data , right ?  if introduce a new func seems they are doing same things